corehq/messaging/smsbackends/vertex/forms.py

```diff
@@ -0,0 +1,34 @@
+from corehq.apps.sms.forms import BackendForm
+from django import forms
+from crispy_forms import layout as crispy
+from django.utils.translation import ugettext_lazy, ugettext as _
+
+
+class VertexBackendForm(BackendForm):
+    username = forms.CharField(
+        label=ugettext_lazy("username"),
+        required=True,
+    )
+    password = forms.CharField(
+        label=ugettext_lazy("password"),
+        required=True,
+    )
+    senderid = forms.CharField(
+        label=ugettext_lazy("senderid"),
+        required=True,
+    )
+    response = forms.ChoiceField(
```

Comment:
this shouldn't be a configurable option - since we're depending on the return value to be present from the api response we should just always send this value as Y